apiary.apib

```diff
@@ -2937,6 +2937,94 @@ Update a milestone.
 
 + Response 204 (application/json;charset=utf-8)
 
+# Group Tasks
+
+## Tasks [/tasks]
+
+### tasks.list [GET /tasks.list]
+
+Get a list of tasks.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + page (Page, optional)
+        + sort (array, optional)
+            + (object)
+                + field (enum[string], required)
+                    + Members
+                        + due_at
+                + order (enum[string], optional)
+                    + Members
+                        + asc
+            + Default
+                + (object)
+                    + field: `due_at`
+                    + order: `asc`
+
++ Response 200 (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + data (array)
+            + (object)
+                + id: `f29abf48-337d-44b4-aad4-585f5277a456` (string)
+                + description (string)
+                + completed: false (boolean)
+                + completed_at: `2016-02-04T16:44:33+00:00` (string)
+                + due_at: `2016-02-04T16:44:33+00:00` (string)
+
+### tasks.info [GET /tasks.info]
+
+Get information about a task.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + id: `6fac0bf0-e803-424e-af67-76863a3d7d16` (string, required)
+
++ Response 200 (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + data (object)
+            + id: `6fac0bf0-e803-424e-af67-76863a3d7d16` (string)
+            + description (string)
+            + completed: false (boolean)
+            + completed_at: `2016-02-04T16:44:33+00:00` (string)
+            + due_at: `2016-02-04T16:44:33+00:00` (string)
+
+### tasks.create [POST /tasks.create]
+
+Create a new task.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + description (string, required)
+        + due_at: `2016-02-04T16:44:33+00:00` (string, required)
+        + work_type_id: `32665afd-1818-0ed3-9e18-a603a3a21b95` (string, required)
+        + estimated_duration: 3600 (number, optional)
```

Comment:
seconds?

Reply:
Q: this field can be written, but not read?

Reply:
That's coming in other PRs. This PR documents what had already been implemented, since a while, as the Todos API.